neurons/miner.py

```diff
@@ -148,13 +149,13 @@ async def compress_code(self) -> None:
             return
         bt.logging.info(f"Code zip path: {self.code_zip_path}")
 
-    def _compute_model_hash(self, repo_id, model_filename, repo_type):
+    def _compute_model_hash(self, repo_id, model_filename):
         """Compute an 8-character hexadecimal SHA-1 hash of the model file from Hugging Face."""
         try:
             model_path = huggingface_hub.hf_hub_download(
                 repo_id=repo_id,
                 filename=model_filename,
-                repo_type=repo_type,
+                repo_type="model",
```

Comment:
I'm not sure if we even have to use repo_type here since model should be the default one
<img width="641" alt="Screenshot 2025-04-15 at 20 08 19" src="https://github.com/user-attachments/assets/08074172-1355-4792-aab5-9ef6a896a84e" />
but yeah this should work as well